src/wallet/test/coinselector_tests.cpp

```diff
@@ -165,7 +165,9 @@ inline std::vector<OutputGroup>& GroupCoins(const std::vector<COutput>& coins)
 inline std::vector<OutputGroup>& KnapsackGroupOutputs(const std::vector<COutput>& coins, CWallet& wallet, const CoinEligibilityFilter& filter)
 {
     CoinSelectionParams coin_selection_params(/* change_output_size= */ 0,
-                                              /* change_spend_size= */ 0, /* effective_feerate= */ CFeeRate(0),
+                                              /* change_spend_size= */ 0,
+                                              /*change_target=*/ CENT,
```

Comment:
I've been trying to format for clang-tidy since seeing #23545 :sweat_smile: 

I'm not super attached to it though, and I understand that the formatting would be inconsistent here. Happy to add spaces if you feel like it's better

Reply:
nah don't bother